plugin/path_checkouts.go

```diff
@@ -39,19 +377,15 @@ func (b *backend) operationReserveStatus(ctx context.Context, req *logical.Reque
 		return logical.ErrorResponse(`"%s" doesn't exist`, setName), nil
 	}
 	respData := make(map[string]interface{})
+
+	// We don't worry about grabbing read locks for these because we expect this
```

Comment:
Why _not_ just grab a read lock on the set and ensure consistency?

nit: I don't think this is necessarily a human-initiated operation. e.g. a web page/app that users hit to check out creds might frequently be polling status.

Reply:
Will do.